### DISCUSSION TITLE
Emoji 15

### DISCUSSION THREAD

**Discussion by C1710** [C1710/blobmoji#119] [General]
https://github.com/C1710/blobmoji/raw/main/images/blobmoji15.jpg

Luckily, as far as I can see, nothing has changed since the pre-release versions of Emoji 15, so the alpha version can be used directly.
The hyacinth emoji is now using the design [from Noto Emoji](https://github.com/googlefonts/noto-emoji/blob/main/svg/emoji_u1fabb.svg), as is the llama emoji. Both have been color-adjusted.

<hr /><em>This discussion was created from the release <a href='https://github.com/C1710/blobmoji/releases/tag/v15.0'>Emoji 15</a>.</em>

**Comment by C1710** [comment]
There is still an [issue](https://github.com/C1710/blobmoji/issues/111) with number/keycap emojis, but I likely won't have much time to work on it right now.

**Comment by DavidBerdik** [comment]
Is it worth rebuilding my Blobmoji Magisk module to use this version? It sounds like you haven't changed anything, but I want to confirm.

**Reply by C1710** [reply]
I have only changed two emojis in comparison to the alpha version which I published a few months ago. It is different to the Emoji 14 release(s) though.

**Reply by DavidBerdik** [reply]
Alright, cool. I will release an updated version of the module so I can pick up your two changes, but since it's a very small change, I am not going to give it a high priority.
